App.js

```diff
@@ -1,25 +1,66 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * @format
- * @flow
- */
-
-import React from 'react';
-import {StyleSheet, View, Text} from 'react-native';
-
+import React, {useState, useEffect} from 'react';
+import {StyleSheet, View, ActivityIndicator} from 'react-native';
 import {Colors} from 'react-native/Libraries/NewAppScreen';
+import Items from './components/Items/Items';
+import Buttons from './components/Buttons/Buttons';
 
 const App = () => {
+  const [isLoading, setLoading] = useState(true);
+  const [sorts, setSorts] = useState({id: false, author: false});
+  const [data, setData] = useState(null);
+
+  const handleFetchData = () => {
+    setSorts({id: false, author: false});
+    setLoading(true);
+    fetchData();
+  };
+
+  const handleSortById = () => {
+    setSorts({id: true, author: false});
+  };
+
+  const handleSortByName = () => {
+    setSorts({id: false, author: true});
+  };
+
+  async function fetchData() {
+    fetch('https://picsum.photos/v2/list')
+      .then(resp => resp.json())
+      .then(resp => {
+        setLoading(false);
+        setData([...resp]);
+      })
+      .catch(error => {
+        console.log(error);
+      });
+  }
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
   return (
     <View style={styles.body}>
-      <View style={styles.containerList}>
-        <Text>Test</Text>
-      </View>
-      <View>
-        <Text>Buttons</Text>
+      <View style={isLoading ? styles.containerSpinner : styles.containerList}>
+        {isLoading ? (
+          <ActivityIndicator color="blue" size="large" />
+        ) : (
+          <Items
+            data={
+              sorts.author
```

Comment:
It will be better to use just `setData` hook to modify arrays. 
Sorting during rendering is a performance issue. 
`handleSortById` and `handleSortByName ` should modify data array and create new data state.